testInterfaces_test.go

```diff
@@ -24,10 +24,14 @@ const (
 )
 
 func openTestConnConninfo(conninfostr string) (*sql.DB, error) {
+	elog := PDALogger{"DEBUG", ""}
```

Comment:
do we need elog init in every function?

Reply:
Yes, we do. There was a recent change that requires all applications using nzgo driver to init elog, else they will get a segmentation fault when trying to connect to the backend. In this test file, every test function can be independently invoked with a `go test` command, so I had to intit elog everywhere.